python/sglang/srt/mem_cache/hicache_storage.py

```diff
@@ -58,23 +67,31 @@ def batch_get(
         pass
 
     @abstractmethod
-    def set(self, key, value) -> bool:
+    def set(self, 
+            key, 
+            value: Optional[Any] = None, 
+            target_location: Optional[Any] = None,
+            target_sizes: Optional[Any] = None) -> bool:
         """
         Store the value associated with the given key.
         Returns True if the operation was successful, False otherwise.
         """
         pass
 
     @abstractmethod
-    def batch_set(self, keys: List[str], values: List[torch.Tensor]) -> bool:
+    def batch_set(self,
```

Comment:
I noticed that some of the type annotations were removed. Could you share the reason for removing them?